driver/normalizer/normalizer.go

```diff
@@ -23,14 +16,178 @@ var Normalize = Transformers([][]Transformer{
 
 // Preprocessors is a block of AST preprocessing rules rules.
 var Preprocessors = []Mapping{
-	// ObjectToNode defines how to normalize common fields of native AST
-	// (like node type, token, positional information).
-	//
-	// https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast#ObjectToNode
+	// replace "IsEmpty" with "Length: 0" in spans
```

Comment:
Throughout, could you please add brief one-liners to each of the transforms to say what it's generally intended to do? I realize some of these may be fairly standard, but it's hard to decipher the intent from the literal syntax. (This first one says what it's doing—but not why)